### PR TITLE
Pod template support

### PR DESCRIPTION
**Pull request by strieflin**

### CODE REVIEW

File: .pre-commit-config.yaml
Comment (sbckr) [inline .pre-commit-config.yaml:1]: please update copyright

File: README.md [hunk 1]
```diff
@@ -153,12 +155,59 @@ EOF
...
+A fully customized sample generator pod template looks like the following:
+
+```yaml
+apiVersion: klyshko.carbnyestack.io/v1alpha1
+kind: TupleGenerator
+metadata:
+  name: mp-spdz-fake
+spec:
+  template:
+    affinity: # Only place pod on nodes running a Linux OS
```
Comment (sbckr) [inline README.md:190]: the table above defines the field for `Affinity` as  `spec.template.spec.affinity` , but in this example, `affinity` is a member of `template`, not `template.spec`

File: klyshko-operator/api/v1alpha1/tuplegenerationjob_types.go
Comment (sbckr) [inline klyshko-operator/api/v1alpha1/tuplegenerationjob_types.go:1]: please update copyright

**Review by sbckr — COMMENTED** [review]
Requires minor updates of two copyright headers and the documentation in README.md

File: .pre-commit-config.yaml
Comment (strieflin) [reply .pre-commit-config.yaml:1]: Done

File: klyshko-operator/api/v1alpha1/tuplegenerationjob_types.go
Comment (strieflin) [reply klyshko-operator/api/v1alpha1/tuplegenerationjob_types.go:1]: Done

File: README.md (hunk 1 above)
Comment (strieflin) [reply README.md]: Good catch! Fixed.

**Review by sbckr — APPROVED** [review]
LGTM

**Pull request merged**